src/org.opensuse.OCIHelper.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.opensuse.OCIHelper">
+    <signal name="hasConflicts">
+    </signal>
+    <signal name="displayProblemAndSolutions">
+      <arg type="s" direction="out"/>
+      <arg type="s" direction="out"/>
```

Comment:
If I'm not wrong, I think to use a QStringList as a second parameter you'll have to use type="as" here in the second parameter